golang/exercises/typeconversions/stringconverstions.go

```diff
@@ -32,8 +51,8 @@ func TestStringConverstions() {
 		isFailed = true
 	}
 
-	if FloatToString(1/3) != "0.33" {
-		fmt.Println("Failed: stringToFloat")
+	if FloatToString(0.3333) != "0.33" {
```

Comment:
If we convert any of the operands in 1/3 to a float value, we would get a floating point result. 

`FloatToString(1/3.0)` or ` FloatToString(1.0/3)` should do the trick for you.